lib/scroll-through-time.js

```diff
@@ -37,20 +58,32 @@ export default {
   config: {
     autoToggle: {
       title: 'Auto Toggle',
-      description: 'Toggle on start.',
+      description: 'Enabled on Atom startup.',
       type: 'boolean',
       default: true
     },
     direction: {
       title: 'Scroll direction',
+      description: 'Set the scroll direction action. Must be up/down on devices without touchpad or sidescroll.',
       type: 'string',
-      default: 'left',
+      default: 'down',
```

Comment:
can we keep `left` as the default?

Reply:
If that's preferred sure thing.

However in the interest making this package more welcoming to all users I was hoping the default could be changed. Desktop users generally don't have side scrolling so for those users the default option serves no value.

Reply:
I see. The issue is that by changing the default, it will break the workflow of the existing users :/

Reply:
Very fair point! (Didn't even think about how atom manages defaults and that changing this would affect any users with default)